packages/ui-components/src/Icon/Icon.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent } from "react";
+import classNames from "classnames/bind";
+import get from "lodash/get";
+
+import * as Icons from "@cockroachlabs/icons";
+import styles from "./Icon.module.scss";
+
+import objectToClassnames from "../utils/objectToClassnames";
+
+type IconSize = "large" | "medium" | "default" | "small" | "tiny";
+type IconTint =
```

Comment:
Where did these tints come from? Are they all in use? (purple seems like it shouldn't be in use)